File: server/EntityLayer/Catalogo/ModeloEntity.py
from datetime import datetime

class ModeloEntity:
    ModeloId: int
    Nombre :str
    FechaRegistro:  datetime
    CodUsuario : str
    Estado: bool


    def Cargar(_DB: any):
        c = ModeloEntity()
        c.ModeloId = _DB['ModeloId']
        c.Nombre = _DB['Nombre']
        # c.FechaRegistro = _DB['FechaRegistro']
        # c.CodUsuario = _DB['CodUsuario']
        # c.Estado = bool(_DB['Estado'])
        return c

 

File: ProyectoORM/server/src/EntityLayer/ModeloEntity.ts
import { AccionEnum } from "../Lib/AccionEnum";

export class ModeloEntity {
    ModeloId: number;
    Nombre: string;
    FechaRegistro: Date;
    CodUsuario: string;
    EstadoRegistro: boolean;
    Action: AccionEnum;
    // FechaRegistroString: string
    // Cont: number
}
